tests/migration/migration.go

```diff
@@ -3285,7 +3263,11 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 	It("should update MigrationState's MigrationConfiguration of VMI status", func() {
 		By("Starting a VMI")
-		vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+		vmi := libvmi.NewAlpine(
```

Comment:
not the fault of this PR, but I think this test would run just as well with `NewGuestless` which requires a bit less resources than `NewAlpine`